src/sentry/search/events/builder/__init__.py

```diff
@@ -21,6 +21,11 @@
     ProfileFunctionsTimeseriesQueryBuilder,
     ProfileTopFunctionsTimeseriesQueryBuilder,
 )
+from .profile_functions_metrics import (  # NOQA
+    ProfileFunctionsMetricsQueryBuilder,
+    TimeseriesProfileFunctionsMetricsQueryBuilder,
+    TopProfileFunctionsMetricsQueryBuilder,
+)
```

Comment:
we've been trying to reduce barrel files across the codebase as this impacts startup time, test time, makes it very easy to introduce import cycles, and makes it harder to find where something is defined

I'd recommend using an absolute import path and not making the problem worse by expanding this `__init__.py` file

Reply:
@viglia please at least acknowledge this comment before merging